src/Foundation/NSString.cs

```diff
@@ -116,17 +116,33 @@ public static IntPtr CreateNative (string str)
 		}
 
 		public static IntPtr CreateNative (string str, bool autorelease)
+		{
+			return CreateNative (str, 0, str.Length, autorelease);
```

Comment:
`str` can be `null`, `str.Length` -> NRE

Reply:
Fixed